src/simplay-jupyter/src/startPauseButton.ts

```diff
@@ -0,0 +1,54 @@
+import { createButton } from './utils';
+import tippy, { Instance } from 'tippy.js';
+
+export class StartPauseButton {
+  private static TOOLTIP_PLAY = 'Play';
+  private static TOOLTIP_PAUSE = 'Pause';
+
+  public button: HTMLButtonElement;
+  private iconSpan: HTMLSpanElement;
+  private playIcon: string;
+  private tooltip: Instance;
+  
+
+  constructor(
+    playIcon: string,
+    pauseIcon: string,
+    pauseClickCallback: () => void,
+    playClickCallback: () => void,
+  ) {
+    this.playIcon = playIcon;
+    let playing = false;
+    this.iconSpan = document.createElement('span');
+    this.iconSpan.classList.add('simplay-icon');
+    this.iconSpan.innerHTML = this.playIcon;
+    this.button = createButton(this.iconSpan, ['simplay-button']);
+    this.tooltip = tippy(this.button, {
+      // default
+      placement: 'top',
+      content: StartPauseButton.TOOLTIP_PLAY,
+      delay: [300, 50],
+      arrow: false,
+      theme: 'light',
+    });
+    
+    this.button.addEventListener('click', () => {
+      if (playing) {
+        pauseClickCallback();
+        playing = !playing;
+        this.iconSpan.innerHTML = playIcon;
+        this.tooltip.setContent(StartPauseButton.TOOLTIP_PLAY);
+      } else {
+        playClickCallback();
+        playing = !playing;
+        this.iconSpan.innerHTML = pauseIcon;
+        this.tooltip.setContent(StartPauseButton.TOOLTIP_PAUSE);
+      }
+    });
+  }
+
+  public reset() {
+    this.iconSpan.innerHTML = this.playIcon;
+    this.tooltip.setContent(StartPauseButton.TOOLTIP_PLAY);
+  }
```

Comment:
I think this might be the culprit causing the issue where you have to click the button twice after resetting.
This could be fixed by moving playing to the class level, and setting it here again, I think?